chain/indexer/src/streamer/mod.rs

```diff
@@ -85,13 +100,11 @@ async fn build_streamer_message(
         )
         .await?;
 
-        local_receipts.extend_from_slice(&chunk_local_receipts);
-
-        let mut chunk_receipts = chunk_local_receipts;
+        let mut chunk_receipts = chunk_local_receipts.clone();
```

Comment:
I suggest we re-order this `chunk_receipts` with the loop below, so you don't need to clone `chunk_local_receipts` (you just loop over `&chunk_local_receipts` as before.